### DISCUSSION TITLE
How do I apply a theme (colour scheme)?

### DISCUSSION THREAD

**Discussion by davida1992** [pnp/powershell#4365] [Q&A]
So far I've been creating a new site, and using the Invoke-PnPSiteTemplate command to apply the template to the new site. It seems to be working great, everything is created, content is all in the correct places and it's wonderful.

The only thing that it doesn't appear to be applying is the site theme, specifically the colour pallet. If I go to the site directly and go to "Change the look", I can select our corporate colours. Is there a way I can perform this action with one of the pnp commands? I did have a brief look, but I may be overthinking it.

Thank you in advance.

**Comment by JuergenRB** [comment]
The simplest option is to execute the following command:
`Set-PnPWebTheme -Theme <YourTheme>`

See [docu](https://pnp.github.io/powershell/cmdlets/Set-PnPWebTheme.html)

**Reply by davida1992** [reply]
Thank you very much for the info. I've tried this and I get the error message:
`Set-PnPWebTheme: Attempted to perform an unauthorized operation.`

If I choose a theme that doesn't exist then I get a different error, so I think it's accepting the theme name that I'm entering at least. I've reauthenticated and I seem to be able to perform other actions just fine, but applying the theme seems to be a no-go. I've given the App Registration all permissions just to check and confirm that it isn't a permission issue.
